Test/spec/simd/simd_address.wast

```diff
@@ -0,0 +1,97 @@
+(assert_return (invoke "load_data_5" (i32.const 0)) (v128.const i8x16 0x7a 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00))
+
+(assert_return (invoke "load_data_1" (i32.const 65495)) (v128.const i32x4 0x44434241 0x48474645 0x4c4b4749 0x504f4e4d))
+(assert_return (invoke "load_data_2" (i32.const 65495)) (v128.const i32x4 0x44434241 0x48474645 0x4c4b4749 0x504f4e4d))
+(assert_return (invoke "load_data_3" (i32.const 65495)) (v128.const i32x4 0x45444342 0x49484746 0x4d4c4b47 0x51504f4e))
+(assert_return (invoke "load_data_4" (i32.const 65495)) (v128.const i32x4 0x46454443 0x47494847 0x4e4d4c4b 0x5251504f))
+(assert_return (invoke "load_data_5" (i32.const 65495)) (v128.const i32x4 0x0000005a 0x00000000 0x00000000 0x00000000))
+
+(assert_return (invoke "load_data_1" (i32.const 65495)) (v128.const i16x8 0x4241 0x4443 0x4645 0x4847 0x4749 0x4c4b 0x4e4d 0x504f))
+(assert_return (invoke "load_data_2" (i32.const 65495)) (v128.const i16x8 0x4241 0x4443 0x4645 0x4847 0x4749 0x4c4b 0x4e4d 0x504f))
+(assert_return (invoke "load_data_3" (i32.const 65495)) (v128.const i16x8 0x4342 0x4544 0x4746 0x4948 0x4b47 0x4d4c 0x4f4e 0x5150))
+(assert_return (invoke "load_data_4" (i32.const 65495)) (v128.const i16x8 0x4443 0x4645 0x4847 0x4749 0x4c4b 0x4e4d 0x504f 0x5251))
+(assert_return (invoke "load_data_5" (i32.const 65495)) (v128.const i16x8 0x005a 0x0000 0x0000 0x0000 0x0000 0x0000 0x0000 0x0000))
+
+(assert_return (invoke "load_data_1" (i32.const 65495)) (v128.const i8x16 0x41 0x42 0x43 0x44 0x45 0x46 0x47 0x48 0x49 0x47 0x4b 0x4c 0x4d 0x4e 0x4f 0x50))
+(assert_return (invoke "load_data_2" (i32.const 65495)) (v128.const i8x16 0x41 0x42 0x43 0x44 0x45 0x46 0x47 0x48 0x49 0x47 0x4b 0x4c 0x4d 0x4e 0x4f 0x50))
+(assert_return (invoke "load_data_3" (i32.const 65495)) (v128.const i8x16 0x42 0x43 0x44 0x45 0x46 0x47 0x48 0x49 0x47 0x4b 0x4c 0x4d 0x4e 0x4f 0x50 0x51))
+(assert_return (invoke "load_data_4" (i32.const 65495)) (v128.const i8x16 0x43 0x44 0x45 0x46 0x47 0x48 0x49 0x47 0x4b 0x4c 0x4d 0x4e 0x4f 0x50 0x51 0x52))
+(assert_return (invoke "load_data_5" (i32.const 65495)) (v128.const i8x16 0x5a 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00))
+
+(assert_trap (invoke "load_data_5" (i32.const 65496)) "out of bounds memory access")
+
+
+;; Load v128 data with invalid offset
+
+(module
+  (memory 1)
+  (func (export "v128.load_offset_65521")
+    (local v128)
+    (v128.store (i32.const 0) (v128.const i32x4 4294967295 4294967295 4294967295 4294967295))
+    (drop (v128.load offset=65521 (i32.const 0)))
+  )
+)
+(assert_trap (invoke "v128.load_offset_65521") "out of bounds memory access")
+
+(assert_malformed
+  (module quote
+    "(memory 1)"
+    "(func"
+    "  (local v128)"
+    "  (v128.store (i32.const 0) (v128.const i32x4 0 1 2 3))"
+    "  (drop (v128.load offset=-1 (i32.const 0)))"
```

Comment:
@AndrewScheidecker, this test fails because `offset=-1` is treated as an invalid argument in WAVM while in WASM interpreter it is malformed.

Reply:
Thanks for pointing that out. Should be fixed in https://github.com/WAVM/WAVM/commit/c89a595016c65e586f6f683a839b9188c7ec5f81

Reply:
Yeah, it passes now!